src/components/todo/TodoAppContainer.tsx

```diff
@@ -25,7 +25,7 @@ const TodoAppContainer = () => {
     setLoading(true);
     await axiosInstance
       .get("/todos")
-      .then((res) => dispatch(setTodoAction(res.data)));
+      .then((res) => dispatch(setTodoAction(res.data.todos)));
```

Comment:
We are fetching the todos on 6 places, can we make it on one place?
I believe this is a good opportunity to refactor this.

Reply:
Prepared a separate service file for this